app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if len(command) == 3 and command[0] == "mv":
+        _, file_name, new_dir = command
```

Comment:
Use meaningful variable names. Use more descriptive names like `source_file` and `destination_path`